src/content/basics/nodepools/index.md

```diff
@@ -112,17 +112,27 @@ Pay close attention to the workloads being rescheduled on other nodes once nodes
 
 See the [`gsctl delete nodepool`](/reference/gsctl/delete-nodepool/) reference for how to delete a node pool using the CLI.
 
-## Instance distribution
+## On-demand and spot instances {#on-demand-spot}
```

Comment:
I myself failed to find the docs in the past because of that. There was just no real search result for "spot instances".